scheme-lib/rnrs.ss

```diff
@@ -184,6 +189,7 @@
         ;; calling error here can lead to an infinite loop, so we
         ;; generate an unreachable instead.
         (%unreachable)))
+  ;; doesn't handle procedure or cyclic structure equivalence
```

Comment:
I opened #110 for adding support for cyclic structures. For completeness, it's probably good to go ahead and support that, even if it's a bit slow without hash tables.